liquibase-maven-plugin/pom.xml

```diff
@@ -30,11 +58,6 @@
             <groupId>javax.xml.bind</groupId>
             <artifactId>jaxb-api</artifactId>
             <version>2.3.1</version>
-        </dependency>
-        <dependency>
-            <groupId>org.apache.maven</groupId>
-            <artifactId>maven-plugin-api</artifactId>
-            <version>${targetMavenVersion}</version>
```

Comment:
Why was this removed?

Reply:
I think it was done as a part of this commit -> [here](https://github.com/liquibase/liquibase/commit/868d09e7868873358f22ad6127f82fbd1620dc81#diff-56e64c6e32ac9a2a4dd259dd9c572ef3342344f575bc38296d5647c7f5b2c341) to flatten the pom.